rotkehlchen/api/v1/resources.py

```diff
@@ -2147,12 +2156,20 @@ def delete(self, watchers: List[str]) -> Response:
 
 class AssetIconsResource(BaseMethodView):
 
-    get_schema = SingleAssetIdentifierSchema()
+    get_schema = AssetIconSchema()
     patch_schema = SingleAssetIdentifierSchema()
     upload_schema = AssetIconUploadSchema()
 
-    @use_kwargs(get_schema, location='view_args')
-    def get(self, asset: Asset) -> Response:
+    @use_kwargs(get_schema, location='json_and_query')
+    def get(self, asset: Asset, t: str) -> Response:  # pylint: disable=unused-argument
+        """Return the icon for one asset.
```

Comment:
This sounds terrible. I don't really know why you do tihs or what bug you refer to

Reply:
This is done only in this endpoint btw and is explained in the docstring. Avoid the bug is the one linked and I moved the identifier to query args or json